src/bindeps_conda.jl

```diff
@@ -1,5 +1,5 @@
 # This file contains the necessary ingredients to create a PackageManager for BinDeps
-using BinDeps
+import BinDeps
```

Comment:
Because BinDeps also have a bindir function and hence gives a warning about conflicts with an existing identyfier and therefore i changed it. If it causes any trouble we could change the bindir function name.